docs/components/overview.md

```diff
@@ -15,9 +16,10 @@ This section contains product manual content for each component in Camunda Cloud
 - [Optimize](optimize/what-is-optimize.md) - Detailed user guide for working with Optimize to improve your processes by identifying constraints in your system.
 - [Tasklist](tasklist/introduction.md) - Documentation for working with user tasks in Tasklist.
 
-
 :::note Looking for deployment guides?
 
 Deployment guides for Camunda Cloud components are available in the [Self-Managed section](./self-managed/overview.md).
 
-:::
\ No newline at end of file
+:::
+
+[//]:# (From a products perspective, do we want Components to be capitalized or lower case?)
```

Comment:
Good question - as in "Camunda Cloud Components" it would make the most sense capitalized, but I could also see us reserving capital letters for brand names. I don't have strong feelings either way here, but it might be good for you to connect with product marketing or marcomm to get a second opinion.

Reply:
Will follow up with Christopher R. in our meeting this week.